src/Stratis.Bitcoin.IntegrationTests/API/ApiSteps.cs

```diff
@@ -276,6 +347,13 @@ private void the_block_should_contain_the_transaction()
                 .Should().Be(this.transaction.GetHash().ToString());
         }
 
+        private void it_is_rejected_and_user_is_told_to_restore_instead()
+        {
+            this.postResponse.StatusCode.Should().Be(StatusCodes.Status403Forbidden);
+
+            //TODO: this.postResponseText.Should().Be("Use recover - via - extpubkey instead.");
```

Comment:
do not leave uncommented pieces of code.

Reply:
you mean commented, and I agree.